README.md

```diff
@@ -136,14 +136,16 @@ Several debuggers are provided, including [snoop](https://github.com/alexmojaki/
 
 1. Install Python 3.9 and [poetry](https://python-poetry.org/docs/#installation).
 2. Run `poetry install` to install Python dependencies.
-3. Run `poetry run python -m core.generate_static_files`. 
-4. In the `frontend` folder:
+3. Change into the `futurecoder` (top-level project) directory.
```

Comment:
To me this step is implied, but if you think it's really necessary, it needs to come before `poetry install` since that installs the dependencies specified within the `futurecoder` directory.